draft-ajitomi-cose-cose-key-jwk-hpke-kem.md

```diff
@@ -82,18 +81,6 @@ It MUST contain the object consisting of the following three attributes.
 
 The "hkc" parameter can be used with existing "EC" {{RFC7518}} and "OKP" {{RFC8037}} keys and the keys for future post-quantum KEMs.
 
-### Restrictions on the Use of Existing Key Parameters
-
-The restrictions on the use of existing common key parameters in a JWK for HPKE KEM are as follows:
-
-- "alg": The parameter MUST be present and contains one of the following values:
-    - "HPKE-v1-Base"
-    - "HPKE-v1-PSK"
-    - "HPKE-v1-Auth"
-    - "HPKE-v1-AuthPSK"
-- "use": The parameter SHOULD NOT be specified. If specified, it MUST be "enc".
-- "key_ops": The parameter SHOULD NOT be specified. If specified, it MUST include "deriveKey" and/or "deriveBits".
-- etc.
```

Comment:
As I mentioned in the COSE key part, I cannot understand the reason why we have to remove the section.